src/index.js

```diff
@@ -1,27 +1,35 @@
 /**
  * @flow
  */
+import * as emails from './store/emails.json';
+import { formattedMessages } from './store';
 
 // example of generating HTML with js
+function renderNewMessage(newMessage) {
```

Comment:
One thing you might do here is break this up a bit; it seems like you're mixing together "what am I rendering" with "where am I putting it".  I'd also recommend sticking to one approach -- I notice you're using both a template literal as well as appending children to the sidebar itself.  A cleaner approach might be to use template literals for everything (e.g.) and then just set the `innerHTML` property for a single "parent" element.

Reply:
This would also have the benefit of being more performant. Touching the DOM (e.g. `appendChild`) is one of the slowest things you can do because the browser has to redraw the page every time.